v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java

```diff
@@ -260,54 +266,50 @@ private POutput writeAvro(PCollection<Row> outputCollection, Schema schema) {
     org.apache.avro.Schema avroSchema = AvroUtils.toAvroSchema(schema);
     return outputCollection
         .apply(
-            "RowToGenericRecord", MapElements.into(TypeDescriptor.of(GenericRecord.class))
+            "RowToGenericRecord",
+            MapElements.into(TypeDescriptor.of(GenericRecord.class))
                 .via(AvroUtils.getRowToGenericRecordFunction(avroSchema)))
         .setCoder(AvroCoder.of(GenericRecord.class, avroSchema))
-        .apply("WriteToAvro", AvroIO.writeGenericRecords(avroSchema)
-            .to(options.getOutputGcsDirectory())
-            .withSuffix(".avro"));
+        .apply(
+            "WriteToAvro",
+            AvroIO.writeGenericRecords(avroSchema)
+                .to(options.getOutputGcsDirectory())
+                .withSuffix(".avro"));
   }
 
   public PCollection<Row> read(Pipeline pipeline, SchemasUtils schema) {
     switch (options.getInputGcsFileFormat()) {
       case CSV:
-        /*
-         * Step 1: Read CSV file(s) from Cloud Storage using {@link CsvConverters.ReadCsv}.
-         */
         PCollectionTuple csvLines = readCsv(pipeline);
-        /*
-         * Step 2: Convert lines to Json.
-         */
         PCollectionTuple jsons = csvLineToJson(csvLines, schema.getJsonBeamSchema());
 
         if (options.getNonTokenizedDeadLetterGcsPath() != null) {
-          /*
-           * Step 3: Write jsons to dead-letter gcs that were successfully processed.
```

Comment:
Please replace `were successfully` with `were not successfully`

Reply:
Done.